glycojones/privateer
# Update webapp dependencies

# What's Changed

- Updated to gemmi 0.6.5 from 0.5.8
- Added temporary fix for Firefox
- Updated clipper-gemmi version
- Moved from sajson to simdjson 
- Remove gemmi from the file reading in privateer-bind.cpp

## Testing

This is a large pull request which changes a lot of the backend, the following must be tested on Chrome and Firefox at the minimum but also Safari preferably. 
- Search a CIF only (7nyd) file - should work on Chrome and fail on Firefox
- Search a PDB file (5fji) - should work on both
- Upload PDB file - should work on both
- Upload CIF file - should display error message on FireFox and work on Chrome
- Quick tests on database and stats page
